onprc_ehr/resources/web/onprc_ehr/model/sources/PMIC_Services.js

```diff
@@ -0,0 +1,76 @@
+
+/*
+ * Copyright (c) 2013-2014 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+EHR.model.DataModelManager.registerMetadata('PMIC_Services', {
+            allQueries: {
+
+            },
+
+            byQuery: {
+                'study.encounters': {
+                    chargetype: {
+                        defaultValue: 'PMIC',
+                        hidden: true
+                    },
+                    date: {
+                        xtype: 'xdatetime',
+                        extFormat: 'Y-m-d H:i',
+                        defaultValue: (new Date()).format('Y-m-d 8:0')
```

Comment:
Is the '8:0' part a typo? I'm not sure what you're intending to use as the defaultValue

Reply:
defaulting the time to 8am